### PR TITLE
Handle --nats-server on cli correctly

### PR DESCRIPTION
**Pull request by rl-0x000**
The --nats-server value is not being read.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -39,6 +40,14 @@ func main() {
 		config.Clustering.RaftBootstrap = c.Bool("raft-bootstrap-seed")
 		config.Clustering.RaftBootstrapPeers = c.StringSlice("raft-bootstrap-peers")
 
+		// NATS server specified on the command line takes precedence
+		if natsServer := c.String("nats-server"); natsServer != "" {
+			if config.NATS.Servers != nil {
+				fmt.Fprintf(app.Writer, "WARNING: overwriting NATS server(s) specified in config file %#v with command line value %s\n", config.NATS.Servers, natsServer)
+			}
+			config.NATS.Servers = []string{natsServer}
```
Comment (tylertreat) [inline main.go]: We should allow passing in a list of servers from the CLI. For example: 

```
$ liftbridge --nats-server nats://localhost:4222,nats://localhost:4223,nats://localhost:4224
```

File: main.go [hunk 2]
```diff
@@ -39,6 +40,14 @@ func main() {
 		config.Clustering.RaftBootstrap = c.Bool("raft-bootstrap-seed")
 		config.Clustering.RaftBootstrapPeers = c.StringSlice("raft-bootstrap-peers")
 
+		// NATS server specified on the command line takes precedence
+		if natsServer := c.String("nats-server"); natsServer != "" {
+			if config.NATS.Servers != nil {
+				fmt.Fprintf(app.Writer, "WARNING: overwriting NATS server(s) specified in config file %#v with command line value %s\n", config.NATS.Servers, natsServer)
```
Comment (tylertreat) [inline main.go]: We don't need this print since it's expected CLI flags will override config file settings.

File: main.go (hunk 1 above)
Comment (rl-0x000) [reply main.go]: Ah, of course... slaps forehead.

Can we rename the flag to *--nats-servers* to more accurately reflect this and bring it into line with the config file? 

If so, I'll update the doco, etc to match as part of this PR.

File: main.go (hunk 2 above)
Comment (rl-0x000) [reply main.go]: I went back & forth on this as it has bitten me in the past when someone else helpfully made changes to the command line in startup scripts. Less of an issue when an explicit config file cli option is required (versus implicit/'well-known' locations). I'll remove it.

File: main.go (hunk 1 above)
Comment (tylertreat) [reply main.go]: Yes, we should rename the flag to `--nats-servers`. Same goes for the config file setting.

Comment (tylertreat) [reply main.go]: Actually, looks like the config file option is already `servers`, so no changes there.

Comment (rl-0x000) [reply main.go]: Ok, this is done. As per the comments I had to workaround some issues in urfave/cli :-(

**Pull request merged**